boards/samr21-xpro/board.c

```diff
@@ -41,12 +41,15 @@ void board_init(void)
     gpio_init(LED0_PIN, GPIO_OUT);
     LED0_OFF;
 
-    /* initialize the on-board antenna switch */
-    gpio_init(RFCTL1_PIN, GPIO_OUT);
-    gpio_init(RFCTL2_PIN, GPIO_OUT);
-    /* set default antenna switch configuration */
-    board_antenna_config(RFCTL_ANTENNA_DEFAULT);
-
```

Comment:
Why move it down?

Reply:
no reason to run this before `cpu_init()`

Reply:
Can you add a mention in the commit? sorry for the nitpicking

Reply:
Never mind this is bikeshedding